bin/create-scripts/block.js

```diff
@@ -3,7 +3,7 @@ const path = require( 'path' );
 const webpack = require( 'webpack' );
 
 const __rootDir = path.resolve( __dirname, '../../' );
-const entryPath = path.resolve( process.argv[ 2 ] );
+const entryPath = path.resolve( 'client/gutenberg/extensions/', process.argv[ 2 ] );
```

Comment:
🙌

Reply:
I'd rather us leave the hard-coded paths out because I also want people to be able to build from outside of the Calypso directory

```bash
cd ~/my-plugins/raglebart
calypso-sdk build block ./raglebart.js
```

Reply:
> it frees consumers thinking about the structure of blocks inside Calypso

what do you mean?

Reply:
>able to build from outside of the Calypso directory

💯 Agreed! Let's aim for flexibility early on.

What do you think about adding [`yargs`](https://www.npmjs.com/package/yargs) or similar in a follow-up PR quite soon and adding a separate sdk entry-point, so we could continue being flexible:

- Include as many CSS entry points as needed: `--editorStyles=/path/editor.scss --themeStyles=/path/theme.scss`
- Custom output directory with `--output=/path/in/jetpack`
- Build blocks from outside Calypso `--block=/path/to/block.js`
- Start working on things like `--php`, `--es5` in a separate PR
etc.

That would also give us API like this:

`sdk build-block --flags`
or
`sdk some-other-task`

Reply:
I've removed the entry path subdir limitation for now: https://github.com/Automattic/wp-calypso/pull/26258/commits/1498fad2fd930378f35a06218f0304c8724bd331

Reply:
I suggest that we iterate on args in subsequent PRs once we merge the npm script.